src/components/shared/scanner/index.js

```diff
@@ -57,6 +58,9 @@ const Scanner = forwardRef(
       });
     }, []);
 
+    const imageSelectedError = () =>
```

Comment:
```suggestion
    const handleImageSelectedError = () =>
```

i think is better to call functions with an "action" verb. Otherwise they look like a variable or not?